Common/Models/Commands/Command.cs

```diff
@@ -4,6 +4,23 @@ namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Models.Co
 {
     public class Command
     {
+        /// <summary>
+        /// Serialziation deserialziation constructor.
+        /// </summary>
+        internal Command()
```

Comment:
Not necessary. Json.Net will merge the empty List with incoming items. The constructor first gets called then the properties get deserialized.